<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EllipsoidDialog</class>
 <widget class="QDialog" name="EllipsoidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X-Radius</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLCDNumber" name="lcdNumber"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y-Radius</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLCDNumber" name="lcdNumber_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z-Radius</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLCDNumber" name="lcdNumber_3"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EllipsoidDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EllipsoidDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
